src/cli.js

```diff
@@ -191,7 +191,12 @@ function httpHandler(req, res) {
         let isIndexFile = /^\/(index\.html)?(\?|$)/.test(req.url);
         let pkgRoot = path.dirname(__dirname);
         let cwd = process.cwd();
-        let mount = cwd && !fs.existsSync(pkgRoot + req.url) ? cwd : pkgRoot;
+
+        let filePath = req.url.indexOf("?") >= 0 ?
+            req.url.substr(0, req.url.lastIndexOf("?")) :
+            req.url;
```

Comment:
Use `url.parse` method instead of string methods.

https://nodejs.org/en/knowledge/HTTP/clients/how-to-access-query-string-parameters/

Reply:
got , I'm about to handle it.